src/main/java/io/beanmapper/config/Configuration.java

```diff
@@ -467,4 +470,24 @@ public interface Configuration {
     BeanConverterStore getBeanConverterStore();
 
     <S, T> BeanConverter getBeanConverter(Class<S> source, Class<T> target);
+
+    boolean isInDiagnosticsMode();
+
+    int getMappingDepth();
+
+    Configuration getParentConfiguration();
```

Comment:
Waarom hier niet Optional<Configuration> teruggeven?

Reply:
Yes, lijkt me een goede wijziging. Ga ik doorvoeren in de nieuwe Configuration-interface.